edgelet/iotedge/src/check/additional_info.rs

```diff
@@ -260,6 +260,6 @@ impl DiskInfo {
 fn pretty_kbyte(bytes: u64) -> String {
     match Byte::from_unit(bytes as f64, ByteUnit::KiB) {
         Ok(b) => b.get_appropriate_unit(true).format(2),
-        Err(_) => "Could not find bytes".to_owned(),
+        Err(err) => format!("could not parse bytes value: {:?}", err),
```

Comment:
`{}`, not `{:?}`

Reply:
byte_unit::ByteError doesn't impl display, only debug